ui/app/components/ui/popover/popover.stories.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import PopOver from './popover.component'
+import TransactionDetails from '../transaction-details'
+import TransactionActivity from '../transaction-activity'
+import TransactionFees from '../transaction-fees'
+import { text } from '@storybook/addon-knobs/react'
+import Icon from '../icon/icon.component'
+import Tooltip from '../tooltip'
+
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+
+const activities = [
+  'Transaction created with a value of 0.000023 ETH at 09:50 on 8/8/2019.',
+  'Transaction submitted with gas fee of 0 WEI at 09:47 on 8/8/2019.',
+  'Transaction confirmed at 09:45 on 8/8/2019.',
+  'Transaction created with a value of 0.000023 ETH at 09:50 on 8/8/2019.',
+  'Transaction submitted with gas fee of 0 WEI at 09:47 on 8/8/2019.',
+  'Transaction confirmed at 09:45 on 8/8/2019.',
+]
+
+const containerStyle = {
+  width: 800,
+  height: 600,
+  background: 'pink',
+  position: 'relative',
+}
+
+const preventDefault = (event) => {
+  event.preventDefault()
+}
+
+const Chart = () => (
+  <div className="popover__chart"></div>
+)
+
+storiesOf('PopOver', module)
+  .add('PopOver - Approve', () => (
```

Comment:
These stories seem to depict the transactions details popover specifically, rather than the base popover component. I would expect these stories to be simpler - focused on demonstrating what the popover can do, how it scales to different sizes, etc.

Maybe these could stay here as complex examples, but they seem more appropriate to a `TransactionDetailsPopover` component perhaps?